cf_xarray/accessor.py

```diff
@@ -600,6 +684,11 @@ def check_results(names, key):
             successful[k] = bool(measure)
             if measure:
                 varnames.extend(measure)
+        elif k in custom_criteria:
```

Comment:
This looks fine. We should try to refactor this logic, maybe by pushing the `skip` argument down to `_get_all`. cc @malmans2

Reply:
I didn't do anything on this since not sure about the `skip` logic.

Reply:
yes. sorry I didn't make clear that that was a suggestion for the future